tests/functests/MultipleLogManagersTests.cpp

```diff
@@ -36,13 +36,39 @@
 using namespace testing;
 using namespace MAT;
 
-class MultipleLogManagersTests : public ::testing::Test,
-                                 public HttpServer::Callback
+class RequestHandler : public HttpServer::Callback 
+{
+   public:
+    RequestHandler(int id) : count_(0), id_(id){}
+
+    int onHttpRequest(HttpServer::Request const& request, HttpServer::Response& response) override
+    {
+        UNREFERENCED_PARAMETER(response);
```

Comment:
Personal Style Preference Nit:
Instead of marking as `UNREFERENCED_PARAMETER`, can we just comment the variable names out in line 44? Like `onHttpRequest(HttpServer::Request const& /*request*/, HttpServer::Response& /*response*/)`

Reply:
Thanks, have commented the unused argument now.